src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -510,6 +520,16 @@ internal void HandleWorkerInitError(Exception exc)
             PublishWorkerErrorEvent(exc);
         }
 
+        internal void HandleWorkerFunctionLoadError(Exception exc)
+        {
+            _workerChannelLogger.LogError(exc, "Did not receive function load response within functionTimeout.");
```

Comment:
Nit: this log message only applies to the timeout case, not the valid worker function load error case

Reply:
good point. Fixed error message. Exception should have more details.